lib/java/benchmarks-common/src/main/java/org/enso/interpreter/bench/BenchmarksRunner.java

```diff
@@ -27,7 +27,7 @@ public class BenchmarksRunner {
 
   public static void run(String[] args) throws RunnerException {
     if (args.length == 0) {
-      args = new String[] {"--jvmArgs=-Dbench.all=true"};
```

Comment:
Because of this cmd line arg that we used, the JMH runner used to spawn forks only with `-Dbench.all=true` and did not pass any of other args to the current JVM (like `--module-path`, `--add-modules`, etc.). By using `--jvmArgsPrepend`, most of the arguments passed to the forks are inherited from the current JVM process.